src/window/dashboard.ts

```diff
@@ -1,10 +1,18 @@
 import { BrowserView, BrowserWindow } from 'electron';
-import { windowMapping } from '.';
+import { windowMapping, didRestoreWindow } from '.';
 
 const dashboardURL = 'http://127.0.0.1:9080/dashboard/c/local/explorer';
 
+const getDashboardWindow = () => ('dashboard' in windowMapping) ? BrowserWindow.fromId(windowMapping['dashboard']) : null;
+
 export function openDashboard() {
```

Comment:
This function does not handle clicking external links (e.g. links to docs or GitHub page). I assume this is okay?

Reply:
I don't think that's a concern here.. maybe we can review a potential use case to make sure I'm not overlooking something?

Reply:
I poked around the dashboard and couldn't find any links to external addresses. We can probably punt on this until somebody complains? (We don't seem to be doing any SSL cert overrides, so having a page incorrectly opening in the dashboard window shouldn't be fatal.)